sktime/forecasting/compose/_fallback.py

```diff
@@ -112,6 +114,29 @@ def __init__(self, forecasters, verbose=False):
 
         self._anytagis_then_set("requires-fh-in-fit", True, False, self._forecasters)
 
+    def _predict_interval(self, fh, X, coverage):
+        pred_int_capability = True
```

Comment:
instead of this, it is perhaps better - and happening earlier for a user - to use `_anytagis_then_set` in `__init__`, with `capability:pred_int`.

Reply:
So if we do it that way, won't this lead to the below error?

https://github.com/sktime/sktime/blob/b781ba444c69b36405f18c8ced2a4c02414b3310/sktime/forecasting/base/_base.py#L655-L661

It might be confusing to users, since they'd expect a `predict_interval` to possible.

Reply:
If I understand your question correctly, it's not going to reach this function. If your class has `_predict_interval`, it's overriding that of the parent classes (which includes BaseForecaster).